.zuul.yaml

```diff
@@ -0,0 +1,221 @@
+    nodes:
+      - name: controller
+        label: cloud-centos-9-stream-tripleo-vexxhost
+      - name: crc
+        label: coreos-crc-extracted-2-28-0-xxl
+
+- nodeset:
+    name: stf-crc_extracted-ocp414
+    nodes:
+      - name: controller
+        label: cloud-centos-9-stream-tripleo-vexxhost
+      - name: crc
+        label: coreos-crc-extracted-2-30-0-xxl
+
+# Based on the 2-node job cookbook at https://github.com/openstack-k8s-operators/ci-framework/blob/main/docs/source/cookbooks/zuul-job-nodeset.md
+- job:
+    name: stf-base-2node
+    parent: podified-multinode-edpm-deployment-crc
+    abstract: true
+    required-projects:
+      - name: github.com/openstack-k8s-operators/dataplane-operator
+        override-checkout: main
+      - name: github.com/openstack-k8s-operators/infra-operator
+        override-checkout: main
+      - name: github.com/openstack-k8s-operators/openstack-operator
+        override-checkout: main
+      - name: github.com/openstack-k8s-operators/openstack-must-gather
+        override-checkout: main
+    pre-run:
+      - ci/pre-2node.yml
+    vars:
+      cifmw_deploy_edpm: false
+      podified_validation: true
+      cifmw_run_tests: false
+    extra-vars:
+      crc_ci_bootstrap_networking:
+        networks:
+          default:
+            range: 192.168.122.0/24
+            mtu: 1500
+          internal-api:
+            vlan: 20
+            range: 172.17.0.0/24
+          storage:
+            vlan: 21
+            range: 172.18.0.0/24
+          tenant:
+            vlan: 22
+            range: 172.19.0.0/24
+        instances:
+          controller:
+            networks:
+              default:
+                ip: 192.168.122.11
+          crc:
+            networks:
+              default:
+                ip: 192.168.122.10
+              internal-api:
+                ip: 172.17.0.5
+              storage:
+                ip: 172.18.0.5
+              tenant:
+                ip: 172.19.0.5
+
+- job:
+    name: stf-base
+    # defined in: https://review.rdoproject.org/cgit/config/tree/zuul.d/_jobs-crc.yaml
+    parent: stf-base-2node
+    abstract: true
+    description: |
+      Run the stf-run-ci role, and then test stf
+    roles:  # adds in dependent roles i.e. put it in the role path
+      - zuul: github.com/openstack-k8s-operators/ci-framework
+    # These are the additional repos that zuul will clone
+    required-projects:
+      - name: openstack-k8s-operators/ci-framework
+        override-checkout: main
+      - name: github.com/infrawatch/service-telemetry-operator
+        override-checkout: master
+      - name: github.com/infrawatch/smart-gateway-operator
+        override-checkout: master
+      - name: github.com/infrawatch/sg-bridge
+        override-checkout: master
+      - name: github.com/infrawatch/sg-core
+        override-checkout: master
+      - name: github.com/infrawatch/prometheus-webhook-snmp
+        override-checkout: master
```

Comment:
Having these as master is a workaround because the stable-1.5 branch may not be merged.

You should not be overridding the branch for service-telemetry-operator.  It'll check out the current working branch by default.
Overridding it to use master is not testing this change